Mathlib/Algebra/BigOperators/Finsupp.lean

```diff
@@ -240,7 +240,7 @@ theorem map_finsupp_prod [Zero M] [CommMonoid N] [CommMonoid P] {H : Type*}
 #align map_finsupp_sum map_finsupp_sum
 
 /-- Deprecated, use `_root_.map_finsupp_prod` instead. -/
-@[to_additive (attr := deprecated)
+@[to_additive (attr := deprecated) -- 2023-12-30
```

Comment:
could you use the `since` syntax everywhere in the PR? Thanks!

Reply:
I wasn't sure if this would play well with to_additive, but let's try if this works!